app/views/static_pages/about/_overview.html.erb

```diff
@@ -5,7 +5,7 @@
     <div class="grid grid-cols-1 sm:grid-cols-3 gap-x-8 gap-y-16">
       <%= render CardComponent.new(classes: 'col-span-3 lg:col-span-1') do |card| %>
         <% card.with_body(classes: 'text-center') do %>
-          <p class="font-bold text-2xl"><%= User.count %></p>
+          <p class="font-bold text-2xl"><%= User.count.to_s.reverse.gsub(/(\d{3})(?=\d)/, '\\1,').reverse %></p>
```

Comment:
Nit: Rails has a `number_with_delimiter` view helper we can use for this instead:

```suggestion
          <p class="font-bold text-2xl"><%= number_with_delimiter(User.count) %></p>
```